src/reference/supported-platforms.md

```diff
@@ -4,52 +4,53 @@ short-title: Supported platforms
 description: The platforms that Flutter supports by platform version.
 ---
 
-## Support tiers
-
-We define three tiers of support for the platforms on
-which apps built with Flutter might be deployed:
-
-1. **Supported**<br>
-   Google-tested platforms that
-   are automatically tested on every commit
-   by continuous integration testing.
-1. **Best effort**<br>
-   Platforms that we intend to support through
-   coding practices,
-   but are only tested on an ad-hoc basis.
-1. **Unsupported**<br>
-   Platforms that we don't test or support.
-   
-{% comment %}
-**IMPORTANT NOTE**:
-When changing the minimum support version of macOS, Windows, or Linux,
-make sure to make a corresponding change in the get started pages:
-  * /src/get-started/install/macos.md
-  * /src/get-started/install/windows.md
-  * /src/get-started/install/linux.md
-{% endcomment -%}
-
-
-## Deploying Flutter
-
-As of the current stable release,
-support for deploying Flutter apps is shown in the
-following table:
+As of Flutter {{site.appnow.flutter}},
+Flutter supports the following hardware architectures and platform versions
+to run Flutter code.
+
+## Supported hardware architectures
+
+Flutter supports running apps on the following hardware architectures.
+
+{% assign chips = site.data.chipsets %}
+{% assign yes = '<span class="material-symbols" style="color: #158477;">verified</span>' %}
+{% assign no = '<span class="material-symbols" style="color: #D43324">dangerous</span>' %}
+{% assign beta = '<span class="material-symbols" style="color: #13C2AD">gpp_maybe</span>' %}
+{% assign na = '<span class="material-symbols" style="color:
+#DADCE0">do_not_disturb_on</span>' %}
+
+| Platform |   x86   |   x64   |  ARM32  |   ARM64   | RISC-V   |
```

Comment:
FYI, we were told by Microsoft to use the "Windows Arm64" branding instead of "Windows ARM64". It looks like that's the branding used on arm.com as well ([link](https://www.arm.com/architecture/cpu)). Should we rebrand these to `Arm32` and `Arm64`?

Reply:
Sounds good!